packages/jobs/lib/activities.ts

```diff
@@ -23,11 +23,18 @@ import {
     getLastSyncDate
 } from '@nangohq/shared';
 import { getLogger } from '@nangohq/utils/dist/logger.js';
+import { BigQueryClient } from '@nangohq/data-ingestion/dist/index.js';
```

Comment:
```suggestion
import { BigQueryClient } from '@nangohq/data-ingestion';
```

Reply:
Would rather keep it explicit with the file extension I think.